documentation/modules/kafkaconnectors/proc-deploying-kafkaconnector.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-connect.adoc
+
+[id='proc-deploying-kafkaconnector-{context}']
+= Deploying a `KafkaConnector` resource to Kafka Connect
+
+Deploy the example `KafkaConnector` to a Kafka Connect cluster. The example YAML file is configured to create a `FileStreamSourceConnector` to send each line of the Kafka license file as a message in Kafka.
```

Comment:
This is the first time we talk about an example. Maybe we could mention in the **KafkaConnector resources** section that one is provided?  (_the_ example sounds like the reader knows about it)